tvb/simulator/plot/tools.py

```diff
@@ -37,22 +37,31 @@
 
 import numpy
 import scipy as sp
-import networkx as nx
+# import networkx as nx
 from tvb.basic.logger.builder import get_logger
+from tvb.simulator.lab import connectivity
+import scipy.stats
+from sklearn.decomposition import FastICA
```

Comment:
Scipy and Sklearn should never be used for plotting

Reply:
fixed